rpcs3/Emu/Io/PadHandler.h

```diff
@@ -319,6 +319,8 @@ class PadHandlerBase
 protected:
 	static const u32 MAX_GAMEPADS = 7;
 
+	std::array<bool, MAX_GAMEPADS> last_connection_status{{ false, false, false, false, false, false, false }};
```

Comment:
why not using a bitset instead of a bool array

Reply:
For 7 boolean values? Seriously?